### PR TITLE
Remove references to dhc-user

### PR DESCRIPTION
**Pull request by squidboylan**
add documentation for each image's default username

### CODE REVIEW

**Comment by justinknierim** [comment]
Looks good!  I was wondering if it would be possible (or recommended) to have a note in that included file, that existing instances created before X date will continue to use dhc-user (or core for coreos).

**Comment by squidboylan** [comment]
Great suggestion, I've updated the docs according to it. Thanks

File: source/dreamcompute/gettingstarted/common/usernames.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+.. Note::
+
+    Instances created before September 21st, 2016 will continue to have
+    "dhc-user" as the default user, except for CoreOS, which will have "core"
+    as the default user.
+
+* Ubuntu: ubuntu
+* Fedora: fedora
+* Debian: admin
```
Comment (drwahl) [inline source/dreamcompute/gettingstarted/common/usernames.rst]: Debian-7: admin
Debian-8: debian

Gotta keep users on their toes!

Comment (drwahl) [inline source/dreamcompute/gettingstarted/common/usernames.rst]: But apparently what the docs say and what is actually happening is no in line (son of a...).  So ignore this!

**Pull request merged**